src/components/include/application_manager/reset_timeout_handler.h

```diff
@@ -30,41 +30,31 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "sdl_rpc_plugin/commands/hmi/on_ui_reset_timeout_notification.h"
-#include "application_manager/event_engine/event.h"
-#include "interfaces/HMI_API.h"
-
-namespace sdl_rpc_plugin {
-using namespace application_manager;
-
-namespace commands {
-
-namespace hmi {
-
-OnUIResetTimeoutNotification::OnUIResetTimeoutNotification(
-    const application_manager::commands::MessageSharedPtr& message,
-    ApplicationManager& application_manager,
-    rpc_service::RPCService& rpc_service,
-    HMICapabilities& hmi_capabilities,
-    policy::PolicyHandlerInterface& policy_handle)
-    : NotificationFromHMI(message,
-                          application_manager,
-                          rpc_service,
-                          hmi_capabilities,
-                          policy_handle) {}
-
-OnUIResetTimeoutNotification::~OnUIResetTimeoutNotification() {}
-
-void OnUIResetTimeoutNotification::Run() {
-  LOG4CXX_AUTO_TRACE(logger_);
-
-  event_engine::Event event(hmi_apis::FunctionID::UI_OnResetTimeout);
-  event.set_smart_object(*message_);
-  event.raise(application_manager_.event_dispatcher());
-}
-
-}  // namespace hmi
-
-}  // namespace commands
-
+#ifndef SRC_COMPONENTS_INCLUDE_APPLICATION_MANAGER_RESET_TIMEOUT_HANDLER_H_
+#define SRC_COMPONENTS_INCLUDE_APPLICATION_MANAGER_RESET_TIMEOUT_HANDLER_H_
+
+namespace application_manager {
+namespace request_controller {
+class ResetTimeoutHandler {
```

Comment:
@LitvinenkoIra please add description of the class

Reply:
@LuxoftAKutsan done. 
653d522